Common/WALinuxAgent-2.0.16/waagent

```diff
@@ -47,6 +47,7 @@ import zipfile
 import json
 import datetime
 import xml.sax.saxutils
+import distro
```

Comment:
Same comment as above - waagent seems to be set up to be a single file script, so I don't think third-party dependencies would be allowed